verification/curator-service/ui/cypress/integration/components/AppTest.spec.ts

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-undef */
+describe('App', function () {
+    it('Homepage with logged out user', function () {
+        cy.visit('/');
+
+        cy.contains('Linelist');
```

Comment:
ideally we'd not even show this, you need to be a reader at least

Reply:
Coming in next PR :) Just testing existing functionality for this PR.